src/main/resources/org/jenkinsci/plugins/workflow/cps/Snippetizer/index.jelly

```diff
@@ -45,18 +45,13 @@ THE SOFTWARE.
                     <d:tag name="listSteps">
                         <j:forEach var="quasiDescriptor" items="${quasiDescriptors}">
                             <j:set var="descriptor" value="${quasiDescriptor.real}"/>
-                            <f:dropdownListBlock title="${quasiDescriptor.symbol}: ${descriptor.displayName}" staplerClass="${descriptor.clazz.name}" lazy="descriptor,item">
+                            <j:set var="symbol" value="${quasiDescriptor.symbol}"/>
+                            <f:dropdownListBlock title="${symbol}: ${descriptor.displayName}" staplerClass="${descriptor.clazz.name}" lazy="descriptor,item,symbol">
                                 <l:ajax>
                                     <j:set var="it" value="${item}"/>
                                     <j:set var="instance" value="${null}"/>
                                     <j:set var="help" value="${descriptor.helpFile}"/>
-                                    <j:if test="${help != null}">
-                                        <tr>
-                                            <td colspan="3"/>
-                                            <f:helpLink url="${help}"/>
-                                        </tr>
-                                        <f:helpArea/>
-                                    </j:if>
+                                    <f:entry title="&lt;strong&gt;&lt;code&gt;${symbol}&lt;code&gt;&lt;/strong&gt;" help="${help}" />
```

Comment:
I would change this to just be `title="${symbol}"`, or use `Step: ` as a prefix or something if that looks weird. Right now the tooltip for the help icon is literally "Help for feature \<strong>\<code>step name here\</code>\</strong>" in recent versions of Jenkins due to the fix for [SECURITY-1955](https://www.jenkins.io/security/advisory/2020-08-12/#SECURITY-1955).